### PR TITLE
The Basics

### PR DESCRIPTION
**Pull request by aniagaska**
Updated readme, added HTML and CSS documents with typeface, color and link styling.

The next step is layout, the Astoria Digital logotype and button styling

### CODE REVIEW

**Review by bronzehedwick — CHANGES_REQUESTED** [review]
Ania this looks good overall! Can we just clean up the indentation on the HTML and CSS files? VSCode should have a function to properly indent somewhere.

**Comment by aniagaska** [comment]
Thanks @bronzehedwick , I added CSS and HTML extensions to format everything properly. I hope this is better!

File: style.css [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+    }
+    h4 {
+      font-size: 1.25em;
+    }
+    h5 {
+      font-size: 1.25em;
+    }
+    h6 {
+      font-size: 1.25em;
+    }
```
Comment (bronzehedwick) [inline style.css]: Thanks @aniagaska! This is almost there. Just one question...are we missing the closing `}` for the 2 `@media` statements?

**Review by bronzehedwick — CHANGES_REQUESTED** [review]

**Comment by aniagaska** [comment]
@bronzehedwick you were right, I added them!

File: style.css (hunk 1 above)
Comment (aniagaska) [reply style.css]: You were right, I added them!

**Review by bronzehedwick — APPROVED** [review]

**Pull request merged**